lib/gollum/public/gollum/stylesheets/template.scss.erb

```diff
@@ -175,3 +175,31 @@ nav.actions {
    }
 }
 
+/* `gollum`-provided CSS for the note and warning macros. */
+.flash.gollum-note::before {
```

Comment:
I think because, for very long notes, it would be reasonable for the icon to still appear at the top left of the note-container, it would make sense to just manually tweak the `padding-top` or the `top` value of the `.flash` icon area. Below I'll paste some suggested CSS changes.

I think we can also reduce some of the repetition by styling `.flash` by itself, or using another class that is shared between both `.gollum-note` and `.gollum-warning`:

```suggestion
.flash::before {
  display: inline-block;
  position: relative;
  padding-top: 0.5em; /* this value may require some tweaking */
  padding-right: 0.5em;
  width: 24px;
  height: 24px;
}
.flash.gollum-note::before {
  content: url('data:image/svg+xml;utf8,<%= rocticon_css(:info) %>');
}

.flash.gollum-warning::before {
  content: url('data:image/svg+xml;utf8,<%= rocticon_css(:alert) %>');
}
```

Reply:
Thanks @benjaminwil ! Makes sense to reduce repetition in this way. I wasn't quite getting the results I wanted with `padding-top` however, so I've now used `vertical-align: middle` instead. This is what the flash elements look like now:
![Screenshot 2023-05-01 at 19 58 37](https://user-images.githubusercontent.com/571173/235503151-828ab08a-1daf-4401-902d-b12385c22c53.png)

Still not perfect, but pretty good. Suggestions for further improvement welcome of course!